src/hll.c

```diff
@@ -2824,12 +2826,21 @@ hll_set_max_sparse(PG_FUNCTION_ARGS)
 {
     int32 old_maxsparse = g_max_sparse;
     int32 maxsparse = PG_GETARG_INT32(0);
+    StringInfoData sql;
 
     if (maxsparse < -1)
         ereport(ERROR,
                 (errcode(ERRCODE_DATA_EXCEPTION),
                  errmsg("sparse threshold must be in range [-1,MAXINT]")));
 
+    if (Gp_role == GP_ROLE_DISPATCH)
+    {
+        initStringInfo(&sql);
+        appendStringInfo(&sql, "select hll_set_max_sparse(%d);", maxsparse);
+        CdbDispatchCommand(sql.data, DF_NONE, NULL);
```

Comment:
`DF_NONE` means no distributed snapshot, this is intended?

Reply:
In this case, the UDF sets a global variable conf. I guess a distributed snapshot is not needed.